articles/azure/azs-how-create-load-balancer.md

```diff
@@ -0,0 +1,282 @@
+
+6. Select **Availability set** for the **Associated to** option, then select the availability set that you created in the previous section.
+
+7. Click the **+ Add a target network IP configuration** button, select the first VM you created in the previous section and select the network IP to be associated with the back end pool.
+
+8. Repeat step 7 for the second VM.
+
+   ![Add a backend pool](images/azs-browser-add-backend-pool.png)
+
+9. Click **OK**
+
+### Create a health probe
+
+When creating a load balancer, a health probe is used to monitor the status of your application. In Azure Stack, the health probe adds and removes VMs from the load balancer backend pool based on their current status.
+
+1. Navigate to your load balancer by clicking on **All services** in the favourites panel, then selecting **Load balancers** under the networking section.
+
+2. Select the load balancer you created earlier from the list.
+
+3. In the blade for your load balancer, select **Health Probes** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add health probe** blade, enter the following information:
+
+   - **Name** - The name of your health probe.
+
+   - **Protocol** - The protocol that the health probe uses to monitor the status of your VMs.
+
+   - **Port** - The port that the health probe uses to monitor the status of your VMs
+
+   - **Interval** - The number of seconds between probe attempts.
+
+   - **Unhealthy threshold** - The number of consecutive probe failures that must occur before a virtual machine is considered unhealthy and is removed from the pool.
+  
+    ![Add a health probe](images/azs-browser-add-health-probe.png)
+
+6. Click **OK**.
+
+### Create a load balancing rule
+
+A load balancer rule defines how traffic is distributed between VMs. The rule listens for traffic on a frontend IP address, then distributes it among the backend pool.
+
+1. Navigate to your load balancer by clicking on **All services** in the favourites panel, then selecting **Load balancers** under the networking section.
+
+2. Select the load balancer you created earlier from the list.
+
+3. In the blade for your load balancer, select **Load balancing rules** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add load balancing rule** blade, enter the following information:
+
+   - **Name** - The name of your load balancing rule.
+
+   - **Frontend IP address** - Clients communicating with this load balancer on the selected IP address and service will have their traffic routed to the target virtual machine by this NAT rule.
+
+   - **Protocol** - The protocol that the rule redirects traffic for.
+
+   - **Port** - The frontend port that the rule listens on.
+
+   - **Backend port** - You can choose to route traffic to the virtual machines in the backend pool using a different port than the one clients use to communicate with the load balancer.

```

Comment:
back-end (if the field label in the UI is Backend leave it as is to identify the label, but change to back-end within the text that follows)

Reply:
Fixed